src/devtools/client/webconsole/reducers/ui.ts

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 import { createSlice, createNextState } from "@reduxjs/toolkit";
 
-interface WebconsoleUIState {
+export interface WebconsoleUIState {
```

Comment:
This is the smallest nit but what's your take on camel case for acronyms — WebconsoleUIState, or WebconsoleUiState?

Reply:
Uh... I've never had to formally think about this before? :)

https://stackoverflow.com/questions/15526107/acronyms-in-camelcase looks like it has some good discussion

I've fallen towards `userId` as of late, although "ID" isn't an acronym.

Microsoft's advice suggests capitalizing both letters in a two-letter acronym.

But no hugely strong opinion here.